cmd/cli/cli.go

```diff
@@ -0,0 +1,155 @@
+package cli
```

Comment:
A non-main package in cmd/ is a bit of a surprise. I think this should go elsewhere.

Reply:
I'll put it in brimsec/zq/cli